### DISCUSSION TITLE
测试一下能不能发图

### DISCUSSION THREAD

**Discussion by Furina-deFontaine** [share121/inter-knot#4979] [其他]
![121516908_p0](https://github.com/user-attachments/assets/972ff92a-ebff-40d3-968f-dbcfec56fcfb)
111

**Comment by Furina-deFontaine** [comment]
看样子得1m以下才行

**Comment by March-777th** [comment]
看饿了<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by Shenjing142** [comment]
图很棒，怎么保存？

**Reply by Furina-deFontaine** [reply]
从网页端点开可以保存

**Comment by passbyDFX** [comment]
好看，孩子爱看

**Comment by manyshoes** [comment]
好看
